src/SMT.hs

```diff
@@ -96,7 +156,7 @@ data Model = Model
 --    - Asserts that the postcondition cannot be reached
 --   If this query is unsatisfiable, then there exists no case where the postcondition can be violated.
 mkPostconditionQueries :: Claim -> [Query]
-mkPostconditionQueries (B behv@(Behaviour _ _ _ (Interface ifaceName decls) preconds postconds stateUpdates _)) = mkQuery <$> postconds
+mkPostconditionQueries (B behv@(Behaviour _ Pass _ (Interface ifaceName decls) preconds postconds stateUpdates _)) = mkQuery <$> postconds
```

Comment:
Why do we care about `Pass`?

Reply:
This was a bug, postconditions should not hold for `Fail` queries (and actually thinking about it now, probably we shouldn't even include them when we split the act into pass / fail specs?).